helm-chart/README.rst

```diff
@@ -56,3 +56,49 @@ To run tests on the deployment, use
 .. code-block:: console
 
     $ helm test --cleanup renku
+
+
+Upgrading
+---------
+Most information related to upgrading from one chart version to another is covered
+in the `values changelog file<https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/values.yaml.changelog.md>`_.
+For upgrades that require some steps other than modifying the values files to be
+executed, we add some instructions here.
+
+Upgrading to 0.8.0
+******************
+This upgrade comes with an upgrade of the keycloak chart from `4.10.2` to `9.8.1`! For
+details on this upgrade check the dedicated section in the
+`the keycloak chart docs<https://github.com/codecentric/helm-charts/tree/master/charts/keycloak#upgrading>`_
+and the `keycloak docs<https://www.keycloak.org/docs/latest/upgrading/>`_.
+
+- Before starting, make sure to check out `the values changelog for this upgrade<https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/values.yaml.changelog.md#upgrading-to-renku-080-includes-breaking-changes>`_
+  and update your values file accordingly.
+
+- The upgrade of keycloak will perform an **irreversible database migration**. It is therefore recommended
+  to **back up your postgres volume** before performing this upgrade.
+
+- **Warning: Persist keycloak-related secrets!**
+
+  If `global.keycloak.postgresPassword.value` and `global.keycloak.postgresPassword.value`
+  have not been explicitly defined in the values file (and thus have been autocreated by helm),
+  add them to the values file now.
+
+  * Get the `keycloak-postgres-password` from the `renku-keycloak-postgres` secret and add it as `global.keycloak.postgresPassword.value`.
```

Comment:
@ableuler this is probably just me... But I overlooked here that the revised section in my values file should be:
```
global:
  keycloack:
    postgressPassword:
      value: actual_password_value
```

And not 
```
global:
  keycloack:
    postgressPassword: actual_password_value
```

Not sure if it is worth adding this as an example here. I could not make a code suggestion that included a multiline code block inside of it.